packages/beacon-node/src/chain/bls/jobItem.ts

```diff
@@ -71,11 +76,13 @@ export function jobItemWorkReq(job: JobQueueItem, format: PointFormat, metrics:
       // and not a problem in the near future
       // this is monitored on v1.11.0 https://github.com/ChainSafe/lodestar/pull/5912#issuecomment-1700320307
       const timer = metrics?.blsThreadPool.signatureDeserializationMainThreadDuration.startTimer();
-      const signatures = job.sets.map((set) => bls.Signature.fromBytes(set.signature, CoordType.affine, true));
+      const signatures = job.sets.map((set) => {
+        const sig = Signature.deserialize(set.signature, CoordType.affine);
```

Comment:
it's worth to add `signatureFromBytes()` util vs the current `signatureFromBytesNoCheck()` util